### PR TITLE
Fixed Linux Background Problems, because of Acrylic Border

### LINKED ISSUE #251: [BUG] Background colors behaving weirdly, all options having hight transparency
**Describe the bug**
every time i open the app the background is on a grey color that is extremely uncomfortable for me, and when i try to change it with the "window background color" option it turns uncomfortably transparent, independent of the color i choose, and on restarting the app it will always start on the previously mentioned gray coloring
also, the "Use song background image" isn't working too

**To Reproduce**
1. Open the app
2. Click on "Settings"
3. Scroll down to window background color
4. change it to any option
5. See the weird transparency

**Expected behavior**
I expected the app to open always with the currently selected background color, and to not have any transparency on options other than the "Transparent" one

**Screenshots**
https://github.com/Founntain/osuplayer/assets/99212613/3c475ffc-a317-46cb-a78a-44cfb2c257ee

**Desktop:**
 - OS: Linux (endeavourOS)
 - Kernel Version: 6.6.2-arch1-1

**Additional context**
I don't know if that gray color is just what "none" is supposed to look like, but still

This started after the [2023.1118.1](https://github.com/Founntain/osuplayer/releases/tag/2023.1118.1) update, and still persists after [2023.1122.0](https://github.com/Founntain/osuplayer/releases/tag/2023.1122.0)

### PR DESCRIPTION
**Pull request by Founntain**
Fixes #251 by using a normal `Border` instead of the `ExperimentalAcrylicBorder` as this isn't supported on Linux. Also disables other `ExperimentalAcrylicBorder` in the MainWindow

### CODE REVIEW

File: OsuPlayer/Windows/FluentAppWindowViewModel.cs [hunk 1]
```diff
@@ -78,6 +79,9 @@ public float BackgroundBlurRadius
         set => this.RaiseAndSetIfChanged(ref _backgroundImage, value);
     }
 
+    public bool IsNonLinuxOs => !RuntimeInformation.IsOSPlatform(OSPlatform.Linux);
+    public bool IsLinuxOs => RuntimeInformation.IsOSPlatform(OSPlatform.Linux);
```
Comment (Cesan) [inline OsuPlayer/Windows/FluentAppWindowViewModel.cs:83]: Can you make these two a one-time evaluated property instead?

**Review by Cesan — CHANGES_REQUESTED** [review]

File: OsuPlayer/Windows/FluentAppWindowViewModel.cs (hunk 1 above)
Comment (Founntain) [reply OsuPlayer/Windows/FluentAppWindowViewModel.cs:83]: But then we need to negate them with a converter for Example if we only have `IsLinuxOs` we need to use a converter for the Visibility when it is a non Linux OS

Comment (Cesan) [reply OsuPlayer/Windows/FluentAppWindowViewModel.cs:83]: No this is not what I meant

Comment (Founntain) [reply OsuPlayer/Windows/FluentAppWindowViewModel.cs:83]: Ohh I see now instead of Expression (=>)

Assigning those values in the `ctor`

Comment (Cesan) [reply OsuPlayer/Windows/FluentAppWindowViewModel.cs:83]: You can just use = {get;} = xxx;

Comment (Founntain) [reply OsuPlayer/Windows/FluentAppWindowViewModel.cs]: I know, but not a fan of using Methods and Expressions in Non Expression Bodies. `ctor` seems like a better place for that

**Review by Cesan — APPROVED** [review]
Now good

**Pull request merged**